voluncheer/registration/tests.py

```diff
@@ -0,0 +1,13 @@
+from django.test import TestCase
```

Comment:
`voluncheer/registration/...` no longer exists, please rebase the commit to reflect the updated code base.